src/panels/experimental-ui/hui-entities-vertical-card.js

```diff
@@ -0,0 +1,118 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import computeDomain from '../../common/entity/compute_domain.js';
+import computeStateDisplay from '../../common/entity/compute_state_display.js';
+import computeStateName from '../../common/entity/compute_state_name.js';
+
+import '../../components/entity/state-badge.js';
+import '../../components/ha-card.js';
+
+import EventsMixin from '../../mixins/events-mixin.js';
+import LocalizeMixin from '../../mixins/localize-mixin.js';
+
+const VALID_DOMAINS = ['binary_sensor', 'light', 'sensor'];
```

Comment:
I don't think this is up to us. Let's just allow anything and people can see if it doesn't work?